app/services/idv/session.rb

```diff
@@ -77,7 +77,7 @@ def create_profile_from_applicant_with_password(user_password)
     end
 
     def deactivation_reason
```

Comment:
I think the crux of this change is going to come in this class and in `gpo_verify_form`. Currently we deactivate the user for in-person by explicitly setting `deactivation_reason` ([ref](https://github.com/18F/identity-idp/blob/3fc7f043d311173ab82e81bf8c0cacb1e4366cf3/app/services/idv/profile_maker.rb#L17) and [ref](https://github.com/18F/identity-idp/blob/3fc7f043d311173ab82e81bf8c0cacb1e4366cf3/app/forms/gpo_verify_form.rb#L25)).

We should make those into something like `#deactivate_for_in_person` or something similar. That was we can change the writes in that single method which will make it easier to change the way they work.

We already have a `activate_after_passing_in_person` method on the profile which appears to be used properly.

@theabrad also made a note on his spike ticket about scheduling the in-person enrollments ([ref](https://cm-jira.usa.gov/browse/LG-10388?focusedId=1662411&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1662411)). We should make sure the deactivate method is associated with the `#schedule_in_person_enrollment` call on the `EnrollmentHelper`